tests/manager/manager-executeReadWriteCommand-003.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+MongoDB\Driver\Manager::executeReadWriteCommand() read and write concern inheritance
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_replica_set(); ?>
+<?php skip_if_not_enough_data_nodes(2); /* w:2 */ ?>
+<?php skip_if_server_version('<', '3.6'); /* readConcernLevel:available */ ?>
+<?php skip_if_not_clean(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+require_once __DIR__ . "/../utils/observer.php";
+
+$manager = new MongoDB\Driver\Manager(URI, ['readConcernLevel' => 'local', 'w' => 2, 'wtimeoutms' => 10]);
+
+$command = new MongoDB\Driver\Command([
+    'aggregate' => COLLECTION_NAME,
```

Comment:
Note: this test [failed in Evergreen](https://spruce.mongodb.com/task/mongo_php_driver_mongo_40_php7__versions~4.0_php_versions~7.3_os_php7~rhel70_test_test_replicaset_auth_patch_c81330c0778a1bd6d314ad3fd7c5d397d5df1dcb_5f1f906e57e85a27ec5e8dec_20_07_28_02_41_52/logs) because MongoDB 4.0.x (and possibly 3.6.x, although I didn't test) requires `aggregate` to include an `$out` stage if the `writeConcern` option is used. Later server versions seem to have relaxed this restriction. Thankfully, our replica set auth environment on Evergreen caught this.